packages/wonder-blocks-announcer/src/clear-messages.ts

```diff
@@ -0,0 +1,15 @@
+import Announcer from "./announcer";
+
+/**
+ * Public API method to clear screen reader messages after sending.
```

Comment:
**nit:** _very_ minor - I think we can remove the "Public API method" part in the comment since this comment block will be shown in code hints in text editors! This is more useful for us so maybe it could be a file comment?